justfile

```
@@ -306,6 +316,11 @@ destroy platform=default_platform:
         ;;
         "AKS-PEER-SNP")
             nix run -L .#terraform -- -chdir=infra/azure-peerpods destroy
+
+            # Clean-up cached image ids.
+            rm -f ${CONTRAST_CACHE_DIR}/*.image-id
```

```suggestion
            rm -f ${CONTRAST_CACHE_DIR}/image-upload/*.image-id
```
nit: Maybe we could have this in a separate directory, similar to the kds cache.